File: ProyectoPoo/ProyectoPooJava/src/ec/edu/utpl/semana9/RunFraccion.java
package ec.edu.utpl.semana9;

public class RunFraccion {
    public static void main(String[] args) {
        Fraccion f1 = new Fraccion(7,1);
        Fraccion f2 = new Fraccion(5,8);
        Fraccion division = f1.dividir(f2);
        System.out.println(f1+" % "+f2+" = "+division);
    }
}

File: ProyectoPoo/ProyectPooKotlin/src/ec/edu/utpl/semana9/RunFraccion.kt
package ec.edu.utpl.semana9

object RunFraccion {
    @JvmStatic
    fun main(args: Array<String>) {
        val f1 = Fraccion(7, 1)
        val f2 = Fraccion(5, 8)
        val division = f1.dividir(f2)
        println("$f1 % $f2 = $division")
    }
}